libgadget/timestep.c

```diff
@@ -30,14 +30,11 @@ static inline inttime_t dti_from_timebin(int bin) {
     /*Casts to work around bug in intel compiler 18.0*/
     return bin > 0 ? (1u << (unsigned) bin) : 0;
 }
-/*Flat array containing all active particles*/
-int NumActiveParticle;
-int *ActiveParticle;
 
-static inline int get_active_particle(int pa)
+static inline int get_active_particle(const ActiveParticles * act, int pa)
```

Comment:
This is an index in P? Do we have an acronym for particle index in P?

Reply:
This is an index in act, but sometimes when act is p, it is the same thing